app/modules/passthroughs/resources.py

```diff
@@ -163,10 +163,14 @@ def get(self, target, path):
         params = {}
         params.update(request.args)
         params.update(request.form)
-
-        response = current_app.sage.request_passthrough(
-            'passthrough.data', 'get', {'params': params}, path, target
-        )
+        if target == 'jobs':
```

Comment:
This feels like a complete hack. Any better suggestions gratefully received but also see discssion in dev-houston.
I think this interface should be restricted and specific rather than unrestricted and generic.